src/Docker/Manager/ImageManager.php

```diff
@@ -5,12 +5,12 @@
 use Docker\Exception\ImageNotFoundException;
 use Docker\Exception\UnexpectedStatusCodeException;
 use Docker\Image;
-use GuzzleHttp\Client;
```

Comment:
To be consistent with ContainerManager and as there is a Docker client as well it’s more clear.